Packs/RedLock/Integrations/RedLock/README.md

```diff
@@ -1,799 +1,419 @@
-<hr>
-<h3 id="h_696049476181530796105219">2. Get RedLock alert details</h3>
-<p>Get details for RedLock alerts.</p>
-<h5>Base Command</h5>
-<p><code>redlock-get-alert-details</code></p>
-<h5>Input</h5>
-<table style="height: 271px; width: 744px;" border="2" cellpadding="6">
-<tbody>
-<tr>
-<td style="width: 210px;"><strong>Input Parameter</strong></td>
-<td style="width: 535px;"><strong>Description</strong></td>
-</tr>
-<tr>
-<td style="width: 210px;">alert-id</td>
-<td style="width: 535px;">Alert ID</td>
-</tr>
-<tr>
-<td style="width: 210px;">detailed</td>
-<td style="width: 535px;">Enables retrieving the entire or trimmed alert model</td>
-</tr>
-</tbody>
-</table>
-<p> </p>
-<h5>Context Output</h5>
-<table style="height: 271px; width: 750px;" border="2" cellpadding="6">
-<tbody>
-<tr>
-<td style="width: 180px;"><strong>Path</strong></td>
-<td style="width: 565px;"><strong>Description</strong></td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.ID</td>
-<td style="width: 565px;">ID of returned alert</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Status</td>
-<td style="width: 565px;">Status of returned alert</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.AlertTime</td>
-<td style="width: 565px;">Time of alert</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Policy.ID</td>
-<td style="width: 565px;">Policy ID</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Policy.Name</td>
-<td style="width: 565px;">Policy name</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Policy.Type</td>
-<td style="width: 565px;">Policy type</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Policy.Severity</td>
-<td style="width: 565px;">Policy severity</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.Policy.Remediable</td>
-<td style="width: 565px;">Whether or not the policy is remediable</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.RiskDetail.Rating</td>
-<td style="width: 565px;">Risk rating</td>
-</tr>
-<tr>
-<td style="width: 180px;">Redlock.Alert.RiskDetail.Score</td>
-<td style="width: 565px;">Risk score</td>
-</tr>
-</tbody>
-</table>
-<p> </p>
-<h5>Command Example</h5>
-<p><code>!redlock-get-alert-details alert-id="P-120"</code></p>
-<h5>Raw Output</h5>
-<pre>{
-	"AlertTime": 1527208131469,
-	"ID": "P-120",
-	"Policy": {
-		"ID": "c2b84f89-7ec8-473e-a6af-404feeeb96c5",
-		"Name": null,
-		"Remediable": false,
-		"Severity": null,
-		"Type": "config"
-	},
-	"Resource": {
-		"Account": "Adrians AWS account",
-		"AccountID": "961855366482",
-		"ID": "arn:aws:cloudtrail:us-west-1:961855366482:trail/Logs",
-		"Name": "Logs"
-	},
-	"RiskDetail": {
-		"Rating": "C",
-		"Score": 20
-	},
-	"Status": "dismissed"
+Cloud threat defense
+This integration was integrated and tested with version xx of RedLock
```

Comment:
update the `xx` to valid version if there is, otherwise delete.
update integration name from `Redlock` to `Prisma Cloud (RedLock) ` in readme

Reply:
Didn't find a version.  Updated to Prisma Cloud (RedLock) where referenced